test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/Infrastructure/TestDefaultValues.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication.Tests.OpenIdConnect.Infrastructre
```

Comment:
I do plan to move move stuff in the this folder. For example the `Transaction` and `TestServer` code. These classes will be shared among test classes. I do intend to have multiple test classes each have limited number of test methods. that improve readability.

Let's keep this conversation when my next PR is out, we can then iron out how the utility class layout.